docs/device_support_guide.md

```diff
@@ -163,3 +165,39 @@ To read EC memory in text form run:
 To save EC memory in text form run:
 
 + `sudo dd if=/dev/mem bs=1 skip=4227860480 count=256 | hexdump -C > ec_dump.txt`
+
+# Contribution
+
+## Firmware naming
```

Comment:
this is a solid explanation , but it's better to put it in another documentation

Reply:
We probably don't need more then 1 guide now

Reply:
These should probably be in a separate guide.

Some people have been requesting a guide on adding a config to the code and I think it also doesn't belong in this file.

We can make a file with a "Contribution" guide that:

1. refers people to this guide if they want to contribute by discovering their laptops' configs (this is a separate file because it is big)
2. provides some specific info like FW naming and generations
3. explains how to add a config to the code